src/client/src/models/Scene.js

```diff
@@ -357,9 +366,8 @@ const Scene = {
   },
 
   loadSceneListFromFile: function(loadedData) {
-	localStorage.setItem('file-name', JSON.stringify({
-      'file-name': loadedData['file_name']
-    }))
+	localStorage.setItem('file-name', JSON.stringify(loadedData['file_name']))
```

Comment:
Shouldn't this be a call to `saveFileName`?

Reply:
No, that's not how our save functions work. Our save functions work with forms, which update the member variables we declare in Scene.js. You don't pass it in a string to do that.